docs/testing/en/FRONTEND_TEST_INDEX.md

```
@@ -33,7 +33,7 @@ and are **not** added again to the grand total.
 ### Feature-Specific Tests (280 tests)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)